course/versioning.py

```diff
@@ -204,6 +213,23 @@ def clean_git_source(self):
 
         return self.cleaned_data["git_source"]
 
+    def clean_force_lang(self):
+        force_lang = self.cleaned_data["force_lang"]
+
+        if not force_lang:
```

Comment:
Code duplicated with below. Maybe introduce a widget to share it?

Reply:
That's a good idea.

Reply:
It seems that should be done by a `validators` param in the the field definition, not widget.

Reply:
Done.